doc/system-administration.md

```diff
@@ -0,0 +1,96 @@
+# System administration and troubleshooting
+
+The goal of this document is to provide enough information to allow a system administrator to maintain a production DRIVER system, and to aid in troubleshooting if something goes wrong. This assumes a production system has already been deployed using the default configuration.
+
+
+## General system architecture
+
+The default DRIVER production setup consists of three servers, all running Ubuntu 14.04 LTS:
+
+1. Database host
+   * `PostgreSQL` database for storing data
+   * `Redis` service for caching temporary data
+2. App host
+   * `Django` application backend
+   * `Nginx` web proxy for routing requests
+   * `Gunicorn` for interfacing between `Nginx` and `Django`
+   * `Windshaft` for serving map tiles
+3. Celery host
+   * `Nginx` web proxy for serving file downloads
+   * `Celery` workers for processing asynchronous and batch requests
+   * `Gradle` service for Android JAR creation
+
+Each of these servers also includes:
+   * `Monit` service for monitoring and restarting services
+   * Firewall configuration via `ufw`
+   * Logs for each locally running service, see [logging.md](logging.md) for details
+
+
+## Deploying updates
+
+Deploying updates to production is done using `Ansible` (version must be at minimum 1.8). An `ansible-playbook` command is run, which uses the local configuration of the application files in the `deployment` directory to deploy updates to the remote servers. It is not necessary to have the application running locally, but it is necessary to have the latest source code, so make sure to pull down the latest version via `git pull`. In addition to the latest source code, there are four files not checked in to the repository that are needed in order to successfully deploy.  These files are as follows
```

Comment:
It'd be good to add a few links to the relevant Ansible documentation pages for group_vars and inventory files.